### PR TITLE
Ran `rake update_fixtures` and updated the specs

### PR DESCRIPTION
**Pull request by boblail**
#### Notes

- drinksmixer.com is no longer on the Internet
- rachaelray.com no longer has the Quick Chick and Noodle Soup so I picked a different recipe
- saveur.com and rachaelray.com started using the JSON+LD microformat

### CODE REVIEW

File: lib/hangry/json_ld_parser.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def parse_recipe_hash
           next unless json['@context'] =~ /schema\.org/
 
           # b/c of www.thespruceeats.com in the specs
-          json = json['mainEntity'] if json.dig('mainEntity', '@type') == 'Recipe'
```
Comment (boblail) [inline lib/hangry/json_ld_parser.rb:39]: This was a bugfix

File: lib/hangry/parsers/non_standard/rachaelray_parser.rb [hunk 2]
```diff
@@ -1,71 +1,23 @@
 module Hangry
   module Parsers
     module NonStandard
-      class RachaelRayParser < RecipeParser
```
Comment (boblail) [inline lib/hangry/parsers/non_standard/rachaelray_parser.rb:4]: Rachael Ray is using JSON+LD now

File: spec/real_examples/saveur_com_spec.rb [hunk 3]
```diff
@@ -3,43 +3,53 @@
 describe Hangry do
   context "saveur.com recipe" do
     before(:all) do
-      skip "saveur.com appears to be using schema.org / RDFa format which will call for a new parser"
```
Comment (boblail) [inline spec/real_examples/saveur_com_spec.rb:6]: Saveur is using JSON+LD now

**Comment by lukeasrodgers** [comment]
Thanks @boblail - been a while since I used this repo, running specs locally I'm getting failures on both this branch and `master` for `spec/real_examples/historical/campbellskitchen_spec.rb`. Do you see those as well?

**Comment by lukeasrodgers** [comment]
I fixed the specs and switched from travis to GH actions for CI here #15

**Comment by boblail** [comment]
> Thanks @boblail - been a while since I used this repo, running specs locally I'm getting failures on both this branch and `master` for `spec/real_examples/historical/campbellskitchen_spec.rb`. Do you see those as well?

Huh! I wasn't seeing any failing specs. I wonder why!

**Pull request merged**